docs/ru/installation.md

```diff
@@ -76,13 +76,21 @@ return [
         ],
 
         /**
-         * Настройка для класса {@see \Yiisoft\Yii\Cycle\Schema\Conveyor\AnnotatedSchemaConveyor}
+         * Настройка для класса {@see \Yiisoft\Yii\Cycle\Schema\Conveyor\MetadataSchemaConveyor}.
          * Здесь указывается список папок с сущностями.
          * В путях поддерживаются псевдонимы {@see \Yiisoft\Aliases\Aliases}.
          */
-        'annotated-entity-paths' => [
+        'entity-paths' => [
             '@src/Entity'
         ],
+        /**
+         * Реализация интерфейса {@see \Yiisoft\Yii\Cycle\Schema\Conveyor\SchemaConveyorInterface},
+         * определяющая источник данных о сущностях:
+         *  - `AnnotatedSchemaConveyor` - парсинг только аннотаций;
+         *  - `AttributedSchemaConveyor` - парсинг только атрибутов (в том числе и на PHP 7.4);
+         *  - `CompositedSchemaConveyor` - парсинг и аннотаций, и атрибутов.
```

Comment:
```suggestion
         *  - `CompositeSchemaConveyor` - парсинг и аннотаций, и атрибутов.
```

Reply:
I support the same style that in Annotated/Attributed Conveyors :)